core/src/main/java/io/grpc/transport/HttpUtil.java

```diff
@@ -69,6 +69,11 @@
   public static final String TE_TRAILERS = "trailers";
 
   /**
+   * The Timeout header name.
+   */
+  public static final String TIMEOUT = "grpc-timeout";
```

Comment:
Ah, I see. That's fine. We could just as easily have the constant in ChannelImpl, though. Either way is fine.